install/firmware.csv

```diff
@@ -1728,17 +1701,11 @@ Trimble Navigation Ltd.,TRIMBLE NETRS,4632120756,1.1-5,2006-10-11T00:00:00Z,2008
 Trimble Navigation Ltd.,TRIMBLE NETRS,4632120756,1.2-0,2008-12-05T14:54:46Z,9999-01-01T00:00:00Z,
 Trimble Navigation Ltd.,TRIMBLE NETRS,4635120781,1.1-5,2006-10-11T00:00:00Z,2009-04-07T15:01:45Z,
 Trimble Navigation Ltd.,TRIMBLE NETRS,4635120781,1.2-0,2009-04-07T15:01:46Z,9999-01-01T00:00:00Z,
-Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.1-5,2006-10-11T00:00:00Z,2008-12-10T14:00:16Z,
-Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.1-5,2008-12-10T14:00:17Z,2008-12-10T14:41:49Z,
-Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.1-5,2008-12-10T14:41:50Z,2008-12-11T12:01:24Z,
-Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.1-5,2008-12-11T12:01:25Z,2009-04-07T18:13:14Z,
+Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.1-5,2006-10-11T00:00:00Z,2009-04-07T18:13:14Z,
 Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.2-0,2009-04-07T18:13:15Z,9999-01-01T00:00:00Z,
```

Comment:
`! Alert ! 'KARA200904070700a.dat': change of receiver firmware from '1.15' to '1.20' ~ 2009 Apr  7 06:59:30.000`

propose to change to:
```
Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.1-5,2006-10-11T00:00:00Z,2009-04-07T06:59:29Z,
Trimble Navigation Ltd.,TRIMBLE NETRS,4635120785,1.2-0,2009-04-07T06:59:30Z,9999-01-01T00:00:00Z,
```